bin/check-disk-usage.rb

```diff
@@ -122,6 +122,13 @@ class CheckDisk < Sensu::Plugin::Check::CLI
          proc: proc(&:to_f),
          default: 100
 
+  option :freespacebelow,
+         short: '-f FREESPACEBELOW',
+         description: 'Trigger alarm only if free space on device is below '\
+           'this threshold (in GB)',
+         proc: proc(&:to_f),
+         default: 10_000
```

Comment:
I would like to better understand where the rationale for this default number came from. I'd imagine this default should be much lower value. 10,000 GB is maybe an appropriate value for a SAN/NAS but I would wager that a double digit number would be more appropriate. Most of my VMs at work have < 100 GB of local storage.